uplink/metainfo/client.go

```diff
@@ -298,11 +310,17 @@ func (client *Client) ListBuckets(ctx context.Context, listOpts storj.BucketList
 	return resultBucketList, nil
 }
 
-func convertBucketToProtoRequest(bucket storj.Bucket) pb.BucketCreateRequest {
+func convertBucketToProtoRequest(bucket storj.Bucket) (bucketReq pb.BucketCreateRequest, err error) {
 	rs := bucket.DefaultRedundancyScheme
+	//TODO @ASK handle err
```

Comment:
is this comment still relevant? or should it be removed?

Reply:
I remember removing it.. apparently not :-).. it should be removed.... removed.